tests/ensemble_evaluator/test_entity.py

```diff
@@ -29,55 +29,55 @@ def _dict_equal(d1, d2):
 def _create_snapshot():
     return (
         SnapshotBuilder()
-        .add_stage(stage_id="0", status="unknown")
-        .add_step(stage_id="0", step_id="0", status="unknown")
+        .add_stage(stage_id="0", status="Unknown")
+        .add_step(stage_id="0", step_id="0", status="Unknown")
         .add_job(
             stage_id="0",
             step_id="0",
             job_id="0",
             name="job0",
             data={},
-            status="unknown",
+            status="Unknown",
```

Comment:
Check if `Unknown` is defined as a non-magic string

Reply:
Not sure what you mean by the comment.

Reply:
Line 40 includes a magic string `"Unknown"`, but it should not be a magic string, it should be a "const".